styles/terminal.less

```diff
@@ -14,6 +14,35 @@ ink-terminal, .ink-terminal {
       background-color: @syntax-background-color !important;
     }
   }
+
+  .search {
+    position: absolute;
+    display: flex;
+    flex-direction: row;
+    top: 0.5em;
+    right: 1em;
+    background-color: @app-background-color;
+    width: 30em;
+    padding: 0.5em;
+    z-index: 5;
+    transition: border-color 0.2s;
+    border: 2px @base-border-color solid;
+    border-radius: 2px;
+
+    >* {
+      padding-left: 0.5em;
+      margin: auto;
+    }
+
+    .searchinput {
+      flex: 1;
+      padding-left: 0;
+    }
+
+    &.nothingfound {
+      border-color: @background-color-error;
+    }
+  }
```

Comment:
Wow, i'm really impressed how little code this took to add! Good job building up abstractions in this repo so reuse was so easy! :) I'm impressed! 😄